pages/glossary/different-hue.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Different hue
+key: different-hue
+unambiguous: true
+objective: true
+---
+
+Two sets of colors have _different hue_ if the hue of any color in one set is different from the hue of any color in the other set.
```

Comment:
We use the word `hue` extensively. Can we link to a basic definition from color theory.

Reply:
I would like that also. Hue is defined here: http://cie.co.at/publications/colour-appearance-model-colour-management-systems-ciecam02
However, from what I can understand that document is only accessible to members of the ICI or something. What I learned is that it's not public. So, I'm not really sure what we can do. Any suggestion?

Reply:
The above link is not accessible to me. (non ICI member). I went through other docs like MDN and color theory etc., no one is cross linking to what hue means, so we should be good.